test/Test/Database/LSMTree/Internal/Merge.hs

```diff
@@ -45,6 +46,7 @@ tests = testGroup "Test.Database.LSMTree.Internal.Merge"
     , testProperty "prop_AbortMerge" $ \level stepSize rds ->
         ioPropertyWithMockFS $ \fs hbio ->
           prop_AbortMerge fs hbio level stepSize rds
+    , testProperty "prop_isStructurallyEmpty" prop_isStructurallyEmpty
```

Comment:
I think it would be nicer to create a new module for this test. This one is for testing the `Merge` datatype.

Reply:
Yep fair enough. Will make a `MergingTree` test module.